gflownet/proxy/uniform.py

```diff
@@ -1,16 +1,16 @@
-from gflownet.proxy.base import Proxy
 import torch
 from torchtyping import TensorType
 
+from gflownet.proxy.base import Proxy
+
 
 class Uniform(Proxy):
```

Comment:
Very nice proxy for testing, I didn't know we had it :) Could we add some readme about these things, somewhat like a guide or a bunch of tips "how to develop and test your env"?

Reply:
Yes, I would like to write up something, but there are so many things to do...